packages/content/src/requests/items/formField/settings/BooleanFormFieldSettings.ts

```diff
@@ -0,0 +1,30 @@
+import { ISerializable, Serializable, type } from "@js-soft/ts-serval";
+
+export interface BooleanFormFieldSettingsJSON {
+    "@type": "BooleanFormFieldSettings";
+}
+
+export interface IBooleanFormFieldSettings extends ISerializable {}
+
+@type("BooleanFormFieldSettings")
+export class BooleanFormFieldSettings extends Serializable implements IBooleanFormFieldSettings {
+    public canCreate(): string | undefined {
+        return;
+    }
+
+    public canAccept(response: string | number | boolean | string[]): string | undefined {
+        if (Array.isArray(response)) {
+            return "Only a selection form field can be accepted with an array.";
+        }
+
+        if (typeof response !== "boolean") {
+            return "The response provided cannot be used to accept the form field.";
+        }
```

Comment:
now as we have brought the validations inside the Settings we can create way better error messages:

```suggestion
        if (typeof response !== "boolean") {
            return "Only a boolean can be used to respond to this item.";
        }
```

same for all other places, the array thing is only needed b/c it all in place before.